### PR TITLE
feat: Add checkbox for disabling leave notification in HR Settings

### PR DESCRIPTION
**Pull request by Anurag810**
image

### CODE REVIEW

**Comment by AfshanKhan** [comment]
@Anurag810 Please resolve conflicts.

**Comment by mohsinalimat** [comment]
@Anurag810 PR title and your repo notification spelling mistake.

**Comment by Anurag810** [comment]
> @Anurag810 Please resolve conflicts.

Done

File: erpnext/hr/doctype/leave_application/leave_application.py [hunk 1]
```diff
@@ -40,7 +40,8 @@ def validate(self):
 	def on_update(self):
 		if self.status == "Open" and self.docstatus < 1:
 			# notify leave approver about creation
-			self.notify_leave_approver()
+			if frappe.db.get_single_value("HR Settings", "send_leave_notification"):
```
Comment (AfshanKhan) [inline erpnext/hr/doctype/leave_application/leave_application.py:43]: Shouldn't this be also added before calling `notify_employee()`

Comment (Anurag810) [reply erpnext/hr/doctype/leave_application/leave_application.py:43]: No. Actually, we are not sending any notification if the checkbox is unchecked, to anyone.

Comment (AfshanKhan) [reply erpnext/hr/doctype/leave_application/leave_application.py:43]: In `on_submit()` `notify_employee()` is called.
Screenshot 2021-03-15 at 4 33 46 PM

It is also called in `on_cancel()`.
Screenshot 2021-03-15 at 4 34 01 PM

And irrespective of if the checkbox is checked or not notification is sent to the employee.
Screenshot 2021-03-15 at 4 35 02 PM

Comment (Anurag810) [reply erpnext/hr/doctype/leave_application/leave_application.py:43]: Got it, I missed it for Leave applier.

**Review by AfshanKhan — APPROVED** [review]

**Pull request merged**